src/app/public/modules/split-view/split-view-workspace-header.component.ts

```diff
@@ -37,9 +41,9 @@ export class SkySplitViewWorkspaceHeaderComponent implements OnDestroy, OnInit {
   constructor(
     private splitViewService: SkySplitViewService,
     private changeRef: ChangeDetectorRef,
-    skyThemeSvc: SkyThemeService
+    @Optional() skyThemeSvc?: SkyThemeService
   ) {
-    skyThemeSvc.settingsChange.subscribe(() => {
+    skyThemeSvc?.settingsChange.subscribe(() => {
```

Comment:
☝️ Not part of the scope of this PR, but this was breaking test fixtures because the arg was not optional.